README.md

```diff
@@ -203,65 +182,49 @@ Example:
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Md Tariq-ul Islam**
 
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: @developertariq
+- LinkedIn: www.linkedin.com/in/developer-tariq
+- Twitter: @developer_tariq
```

Comment:
- [ ] kindly Provide Proper links for your `Github` and `Twitter` accounts to make your readme descriptive. 👍🏻